src/EventStore.ClientAPI/ICompatibilityMode.cs

```diff
@@ -0,0 +1,16 @@
+namespace EventStore.ClientAPI {
+	/// <summary>
+	/// Compatibility mode abstraction.
+	/// </summary>
+	public interface ICompatibilityMode {
+		/// <summary>
+		/// Is EventStoreDB Version 5 compatibility mode enabled.
+		/// </summary>
+		bool IsVersion5CompatibilityModeEnabled();
```

Comment:
Should this be `IsVersion20CompatibilityModeEnabled()` and `IsVersion21CompatibilityModeEnabled()`?